src/integrationTest/java/uk/gov/hmcts/reform/judicialapi/elinks/SchedulerElinksJobIntegrationTest.java

```diff
@@ -61,11 +73,9 @@ void test_load_elinks_job_status_sucess() throws JOSEException, JsonProcessingEx
         dataloadSchedulerJobRepository.deleteAll();
 
         elinksApiJobScheduler.loadElinksJob();
+        List<DataloadSchedulerJob> audits = dataloadSchedulerJobRepository.findAll();
 
-        DataloadSchedulerJob jobDetails = dataloadSchedulerJobRepository.findAll().get(0);
-
-        assertThat(jobDetails).isNotNull();
-        assertThat(jobDetails.getPublishingStatus()).isNotNull();
```

Comment:
Why was this deleted? Is this not required for the test case to be valid? Otherwise all the test case is doing is verifying an empty list

Reply:
We will cover it as a part of  scheduler user story